Extensions/Settings/Dnn.PersonaBar.SiteSettings/SiteSettings.Web/src/components/siteAliasSettings/siteAliases/siteAliasEditor/index.jsx

```diff
@@ -40,27 +35,6 @@ class SiteAliasEditor extends Component {
         }
     }
 
-    componentDidUpdate(props) {
```

Comment:
Removed without alternate solution

Reply:
Removed because redundant. Just a duplicate of **onSettingChange()** with redundant setState call.